shell/platform/android/io/flutter/embedding/engine/loader/FlutterLoader.java

```diff
@@ -60,10 +60,22 @@ public static FlutterLoader getInstance() {
     return instance;
   }
 
+  public FlutterLoader() {
+    this(null);
+  }
+
+  public FlutterLoader(@Nullable FlutterJNI flutterJNILoader) {
```

Comment:
replace all `flutterJNILoader` with just `flutterJNI`. Since there's nothing else called jni loader here.

Reply:
Done, also made this @NonNullable and made the default ctor new one up.